lib/src/signed_video_h26x_common.c

```diff
@@ -928,14 +928,18 @@ update_hash(signed_video_t *self,
 }
 
 static svrc_t
```

Comment:
You should put this before `get_hash_wrapper()`. It is now placed among the different hashing schemes.